authors/apps/authentication/migrations/0002_remove_user_is_staff.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 2.1.5 on 2019-01-16 16:06
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
@VivianDoreen Remove these migration files from your pr. You are not making any model changes.

Reply:
I have made the necessary changes